script.js

```diff
@@ -1,61 +1,57 @@
-// Get elements
-const bookList = document.getElementById('bookList');
-const addButton = document.getElementById('addButton');
-const titleInput = document.getElementById('titleInput');
-const authorInput = document.getElementById('authorInput');
-
-// Initialize book collection from local storage
-let myBooks = JSON.parse(localStorage.getItem('myBooks')) || [];
-
-// Render book list
-function renderBookList() {
-  bookList.innerHTML = '';
-  myBooks.forEach((book, index) => {
-    // Create book item
-    const bookItem = document.createElement('div');
-    bookItem.innerHTML = `
-      <span>${book.title}<br>${book.author}</span><br>
-      <button class="removeButton" data-index="${index}">Remove</button><br><br><hr>
-    `;
-    bookList.appendChild(bookItem);
-  });
+/*  eslint-disable max-classes-per-file, no-unused-vars */
+class Book {
+  constructor(title, author, id) {
+    this.title = title;
+    this.author = author;
+    this.id = id;
+  }
 }
 
-// Add new book
-function addBook() {
-  const title = titleInput.value.trim();
-  const author = authorInput.value.trim();
-
-  // Add book to collection
-  myBooks.push({ title, author });
+class BookManager {
+  static createBookElement(book) {
+    return `
+      <tr id="${book.id}">
+        <td class="book-info"> "${book.title}" by ${book.author}</td>
+        <td class="remove-button">
+          <button type="button" onclick="removeBookWrapper('${book.id}')">Remove</button>
+        </td>
+      </tr>`;
+  }
 
-  // Update local storage
-  localStorage.setItem('myBooks', JSON.stringify(myBooks));
+  static loadData() {
+    const tbody = document.querySelector('tbody');
+    const books = JSON.parse(localStorage.getItem('books')) || [];
+    books.forEach((bookData) => {
+      const book = new Book(bookData.title, bookData.author, bookData.id);
+      tbody.innerHTML += BookManager.createBookElement(book);
+    });
+  }
```

Comment:
- [ ] I noticed that although the books get saved in local storage, they don't get loaded onto the screen when I refresh the page. This is because you forgot to call the static method `loadData` when the page loads. Please make this change 👍

Reply:
Done